controllers/conditions.go

```diff
@@ -106,14 +106,14 @@ func (r *ImageClusterInstallReconciler) setImageReadyCondition(ctx context.Conte
 
 func (r *ImageClusterInstallReconciler) setHostConfiguredCondition(ctx context.Context, ici *v1alpha1.ImageClusterInstall, err error) error {
 	cond := hivev1.ClusterInstallCondition{
-		Type:    hivev1.ClusterInstallStopped,
-		Status:  corev1.ConditionFalse,
+		Type:    hivev1.ClusterInstallRequirementsMet,
```

Comment:
Why change this?

I would understand this change as a part of making BMH required, but does it have anything to do with exiting the loop early?

Reply:
As we talked, missing bmh should not stop installation but rather set the requirements are not yet met